tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -269,9 +269,11 @@ everflow:
 
 everflow/test_everflow_per_interface.py:
   skip:
-    reason: "Skip running on dualtor testbed."
+    reason: "Skip running on dualtor testbed or multi-asic due to https://github.com/sonic-net/sonic-buildimage/issues/11776"
```

Comment:
How about put the issue in conditions?

Reply:
@Blueve - Yes, taken care. Included the issue in conditions.